Automattic/jetpack
# My Jetpack: Adding Connection Screen

Related with #22291 

#### Changes proposed in this Pull Request:

* Adding `ConnectionScreen` component.
* Adding `connection` route.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Goes into `MyJetpackScreen` component
* Insert a `Link` component from `react-router-dom` any place, and set the `to` for `/connection` route.
* Build `My Jetpack` (`jetpack build packages/my-jetpack`).
* Navigate to `Dashboard`
* Disconnect Jetpack
* Reconnect it, but doesn't connect `wpcom` account.
* Navigate to `My Jetpack`.
* Click in the `Link`.
* It should load the `ConnectionScreen` screen.
* Click in `Connect your user account`, it should start connection flow.

#### ⚠️ Layout Warnings

Some points in the current screen are different from `Figma`, basic these points:

- It implements `Go back` instead `X` button, which aligns better with products flow (we can discuss it later).
- `Connect` button is `blue` instead `black`.
- Bottom descriptions don't include the list of services to connect with.
- Missing `And more! See all Jetpack features` list item.

Most of them will be implemented in the upcoming PRs.

#### Demo

https://user-images.githubusercontent.com/1663717/152056297-b6ca26f7-bc39-44db-8108-050c55218efe.mp4
